relayer/gasless.py

```
@@ -67,56 +78,93 @@ def permit(web3, private_key, sender_private_key, input_token, input_amount, spe
+                account.address,
+                spender,
+                input_amount,
+                deadline,
+                v,
+                r,
+                s
+            ).call({'from': spender})
+            print(message)
+            print(f"Permit function call emulation successful with version {version}")
+            break
+        except:
```

Ruff

Do not use bare `except`

(E722)